DigiMeSDK/Core/Classes/Entities/DMEServiceGroup.m

```diff
@@ -0,0 +1,59 @@
+//
+//  DMEServiceGroup.h
+//  DigiMeSDK
+//
+//  Created on 21/11/2019.
+//  Copyright © 2019 digi.me Limited. All rights reserved.
+//
+
+#import "DMEServiceGroup.h"
+
+static NSString * const kServiceGroupId = @"id";
+static NSString * const kServiceTypes = @"serviceTypes";
+
+@interface DMEServiceGroup()
+
+@property (nonatomic, strong, readwrite) NSNumber *serviceGroupId;
```

Comment:
Making these properties `readwrite` is not needed as `readonly` properties can still be set in initialiser.  This comment also applies to `DMEServiceType` and `DMEServiceObjectType`